Storage/Storage/Model/OrderCount+CoreDataProperties.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import CoreData
+
+
+extension OrderCount {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<OrderCount> {
+        return NSFetchRequest<OrderCount>(entityName: "OrderCount")
+    }
+
+    @NSManaged public var siteID: Int64
+    @NSManaged public var items: NSSet?
```

Comment:
This is usually edited by hand. `NSSet?` should be `Set<OrderCountItem>?`

Reply:
🤦‍♂  I edited it manually in the mark III branch, and forgot to back port it here. Sorry!